docs/schemas/README.md

```diff
@@ -12,6 +12,8 @@ The jpo-ode supports receiving and decoding ASN1 messages from RSUs. The support
 
 The output JSON of the ODE is complex but it is similar to the official standard of J2735 with some minor differences due to the form of their deserialized POJOs. To help implement proper data validation for the JSON output of the ODE into any data pipeline infrastructure, you may use the provided validation schemas within this directory.
 
+Each jpo-ode output object also contains a `schemaVersion`. This value represents the version of the jpo-ode output that the object came from. This version is not intended to be changed except in the scenario of newly added breaking changes to the output. This value is set [here](https://github.com/usdot-jpo-ode/jpo-ode/blob/develop/jpo-ode-core/src/main/java/us/dot/its/jpo/ode/model/OdeMsgMetadata.java).
```

Comment:
suggestion(non-blocking): We can use a local path to the file to allow users to navigate to the file locally instead of online
```suggestion
Each jpo-ode output object also contains a `schemaVersion`. This value represents the version of the jpo-ode output that the object came from. This version is not intended to be changed except in the scenario of newly added breaking changes to the output. This value is set [here](./jpo-ode-core/src/main/java/us/dot/its/jpo/ode/model/OdeMsgMetadata.java).
```